<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaServerWindow</class>
 <widget class="QMainWindow" name="MediaServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>382</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>170</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Simple Media Server Test</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="sharedItemsTable">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Recursive</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Monitored</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Path</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item row="10" column="1">
       <widget class="QPushButton" name="addItemButton">
        <property name="text">
         <string>Add Items</string>
        </property>
       </widget>
      </item>
      <item row="10" column="5">
       <widget class="QPushButton" name="DeleteDirectoriesButton">
        <property name="text">
         <string>Delete Selected Directories</string>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QCheckBox" name="scanRecursivelyCheckbox">
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="text">
         <string>Scan recursively</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="4">
       <widget class="QPushButton" name="addContentButton">
        <property name="text">
         <string>&amp;Add New Root Directory</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="hide">
      <property name="text">
       <string>Hide</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
